editor/src/components/navigator/layout-element-icons.ts

```diff
@@ -181,6 +182,39 @@ function createLayoutIconProps(
   return null
 }
 
+function isConditionalBranchText(
+  navigatorEntry: NavigatorEntry | null,
+  metadata: ElementInstanceMetadataMap,
+) {
```

Comment:
Can you add return values to the new functions please

Reply:
uh, sure!